pkg/cmd/cli/cli.go

```diff
@@ -143,6 +143,7 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 				cmd.NewCmdProxy(fullName, f, out),
 				cmd.NewCmdAttach(fullName, f, in, out, errout),
 				cmd.NewCmdRun(fullName, f, in, out, errout),
+				cmd.NewCmdCompletion(fullName, f, out),
```

Comment:
Yes

On Jun 15, 2016, at 4:31 PM, Fabiano Franz <notifications@github.com> wrote:

In pkg/cmd/cli/cli.go
<https://github.com/openshift/origin/pull/9354#discussion_r67239787>:

> @@ -143,6 +143,7 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
>  				cmd.NewCmdProxy(fullName, f, out),
>  				cmd.NewCmdAttach(fullName, f, in, out, errout),
>  				cmd.NewCmdRun(fullName, f, in, out, errout),
> +				cmd.NewCmdCompletion(fullName, f, out),

I think this should go in the "settings" group, @smarterclayton
<https://github.com/smarterclayton> agree?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9354/files/cef5989d3fe809b129a5994a7cf393b7038dd6fc..5f930b7a1faddc06464cc5f383d13bb0b8c917b4#r67239787>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p_cTYUEO2hwK4XPTYN4jwwqnZsEzks5qMGEngaJpZM4I2t-y>
.